packages/erb-formatter/package.yaml

```diff
@@ -0,0 +1,17 @@
+---
+name: erb-formatter
+description: Format ERB files with speed and precision.
+homepage: https://github.com/nebulab/erb-formatter
+licenses:
+  - MIT
+languages:
+  - HTML
+  - Ruby
+categories:
+  - Formatter
+
+source:
+  id: pkg:gem/erb-formatter@0.7.2
+
+bin:
+  erb-format: gem:erb-formatter
```

Comment:
```suggestion
  erb-formatter: gem:erb-formatter
```

Reply:
Hi @williamboman can you please revert this to `erb-format` or is there a way that we can have 2 binaries `erb-format` and `erb-formatter`?

I understand that there might be some confusion because of the difference between the name of the gem and its corresponding binary, but `erb-format` is the correct way to use this gem as documented [here](https://github.com/nebulab/erb-formatter?tab=readme-ov-file#usage).

And other nvim tools like `conform.nvim` are configured to use the `erb-format` binary, as you can see [here](https://github.com/stevearc/conform.nvim/blob/master/lua/conform/formatters/erb_format.lua).

Thanks :)

Reply:
Ah, apologies. It's a common thing that the executable name is incorrect so it's a habit, should've double checked. That seems really odd though. I guess they provide both `erb-format` and `erb-formatter` executables? 7'll update right away

Reply:
Will be fixed in https://github.com/mason-org/mason-registry/pull/5044. They do provide both executables, I guess the latter is an old variant.

Reply:
They do provide both variants, and I guess you are right it must be an old executable and old documentation, and everyone else just ran with it in their tools. Good old legacy issue :)

Thanks for sorting this out, much appreciated.